src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import Waiting from './waiting';
+import {browserHistory} from 'react-router';
+import ApiMixin from '../../../mixins/apiMixin';
+
+import ProjectContext from '../../projects/projectContext';
+import ProjectDocsContext from '../../projectInstall/docsContext';
+import ProjectInstallPlatform from '../../projectInstall/platform';
+
+const Configure = React.createClass({
+  propTypes: {
+    next: React.PropTypes.func
+  },
+
+  mixins: [ApiMixin],
+
+  componentWillMount() {
+    const {platform} = this.props.params;
+    //redirect if platform is not known.
+    if (!platform || platform === 'other') {
+      this.redirectToNeutralDocs();
+    }
+  },
+
+  componentDidMount() {
+    this.timer = setInterval(() => {
+      this.fetchEventData();
+    }, 2000);
+  },
+
+  componentWillUnmount() {
+    clearInterval(this.timer);
+  },
+
+  redirectUrl() {
+    let orgId = this.props.params.orgId;
+    let projectId = this.props.params.projectId;
+    const url = `/${orgId}/${projectId}/#welcome`;
```

Comment:
+1 to follow js's definition of `const`/`let` (variables that get reassigned). Main reason being, it's easier to enforce via eslint.